lnwallet/interface.go

```diff
@@ -636,6 +636,19 @@ func InternalKeyForAddr(wallet WalletController, netParams *chaincfg.Params,
 
 	walletAddr, err := wallet.AddressInfo(addr)
 	if err != nil {
+		// If the error is that the address can't be found, it is not
+		// an error. This happens when any channel which is not a custom
+		// taproot channel is cooperatively closed to an external P2TR
+		// address. In this case there is no internal key associated
+		// with the address. Callers can use the .Option() method to get
```

Comment:
Q: how can the callers get the value, don't we return a `none` option here?

Reply:
I copy-pasted this part of the comment from the check below:

```go
        // No wallet addr. No error, but we'll return an nil error value here,
        // as callers can use the .Option() method to get an option value.
        if walletAddr == nil {
                return none, nil
        }
```

I propose to modify it in both cases to the following: "callers can inspect the returned option to determine if it has a value".

cc @guggero